src/app/components/navbar/navbar.component.ts

```diff
@@ -48,6 +48,11 @@ export class NavbarComponent implements OnInit, OnDestroy {
 			url: '/about',
 			name: 'About',
 			icon: 'fa-info'
+		},
```

Comment:
I don't know if the GPA calculator is a big enough component to put on its own page, though I don't know where else we could put it? We can talk in the Slack

Reply:
Maybe make it a homepage module?